tomcat/src/main/java/org/apache/coyote/http/HttpStatusCode.java

```diff
@@ -3,7 +3,11 @@
 public enum HttpStatusCode {
 
     OK("200 OK"),
-    FOUND("302 Found");
+    FOUND("302 Found"),
+    BAD_REQUEST("400 Bad Request"),
+    NOT_FOUND("404 Not Found"),
+    METHOD_NOT_ALLOWED("405 Method Not Allowed"),
+    INTERNAL_SERVER_ERROR("500 Internal Server Error");
```

Comment:
지난 리뷰와 마찬가지로 code와 message를 따로 관리해보는 것 어떨까요?

```suggestion
    INTERNAL_SERVER_ERROR(500, "Internal Server Error");
```

Reply:
현재 시점에서 둘을 나눠 사용할 필요가 있을까요..?

Reply:
아니요! 사용하는 관점에서는 매번 같이 사용되어서 묶어두는 것도 좋은 방안인 것 같아요. 

다만 저는 둘을 분리했는데 statusCode와 statusMessage의 성격이 다르다고 생각했던 것 같아요. 이후 filter를 추가하는 상황을 생각했을 때에도 statusCode에 대한 정보만 필요한 상황이 머릿속에 그려지기도 했고요..

물론 분리하지 않아도 지금도 충분히 좋은 구조라 생각합니다!